### DISCUSSION TITLE
Is there a way to get a reference to the current line / cursor?

### DISCUSSION THREAD

**Discussion by agjs** [ianstormtaylor/slate#4535] [Q&A]
Hi guys. I've implemented the [forced layout example](https://www.slatejs.org/examples/forced-layout) into my own project and I'm wondering, if there is a way to get a reference to a specific line/cursor?

Essentially, I'd like to achieve something similar to what Medium editor has. Once a user is entering a e.g. Title, I'd like to show a tooltip icon left of that location that a user could hover and get some tips about how to choose a proper title, etc. Respectively, once user moves to the next "forced" location, I'd like to show an icon again showing something else.

![screen](https://user-images.githubusercontent.com/7226555/133843325-c49b9636-7a31-47c4-a449-da56bc25882d.PNG)

In a nutshell, how would I go about getting a reference of where a user is inside of the editor.

Thanks a bunch,
Alex

**Comment by agjs** [comment]
Anyone to help with this, please?

**Comment by dlqqq** [comment]
Have you tried using `editor.selection`?

**Comment by davevilela** [comment]
Try 
`Editor.above(editor, { at: editor.selection })`
